### PR TITLE
feat(about):  aboutUs section

### PR DESCRIPTION
**Pull request by AmandaKr**
## Jira issue

JIRA issue: [FSU-129](https://klau.atlassian.net/browse/FSU-129)

## Describe your changes

This PR introduces the AboutUs section for the About Us page.
## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/About/components/AboutUs/AboutUs.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import TextBanner from "design-system/patterns/textBanner"
+
+import * as Styled from "./AboutUs.styled"
+
+const AboutUs = () => {
+  const theme = useTheme()
+
+  return (
+    <Styled.ContainerSection variant="section">
+      <Styled.Wrapper>
```
Comment (im-klau) [inline packages/frontend/app/templates/About/components/AboutUs/AboutUs.jsx:14]: Comparing to the design, the left column should be smaller and the right one a bit larger.

Preview:
Zrzut ekranu 2023-03-22 o 13 58 22

Design:
Zrzut ekranu 2023-03-22 o 13 58 32

**Review by im-klau — CHANGES_REQUESTED** [review]
Just a few comments 🙌

File: packages/frontend/app/public/home/aboutus-section-image.png
Comment (im-klau) [inline packages/frontend/app/public/home/aboutus-section-image.png:1]: this png should be in a separated directory "about", inside `public`

File: packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    width: 100vw;
+    position: relative;
+  }
+`
+
+export const Img = styled(Image)`
+  width: 100%;
+  height: 700px;
+  object-fit: cover;
```
Comment (im-klau) [inline packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js:44]: the `object-fit: cover` is already applied to the Image component so you can get rid of it here

File: packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  object-position: 40%;
+  margin-top: 200px;
+
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    object-fit: cover;
```
Comment (im-klau) [inline packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js:49]: same with this object-fit

**Review by paulgrym — APPROVED** [review]
Great job! 🌟

File: packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    width: 100vw;
+    position: relative;
+  }
+`
+
+export const Img = styled(Image)`
+  height: 700px;
+  object-position: 40%;
+  margin-top: 200px;
```
Comment (paulgrym) [inline packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js:44]: Oh I jus spotted that in design there is no margin-top. But without it the photo is not heigh enough so I would remove `height:700px;` and add max-width or some grid 🤔

**Review by Ebmili — APPROVED** [review]
Looks great :)

File: packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js (hunk 4 above)
Comment (im-klau) [reply packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js:44]: @AmandaKr just one, additional comment, and once it's resolved we can merge it 🚀

Comment (AmandaKr) [reply packages/frontend/app/templates/About/components/AboutUs/AboutUs.styled.js:44]: @im-klau  I'll correct it before today's daily and will merge it

**Review by nata-kab — APPROVED** [review]
I found nothing to fix 🕵️‍♀️ 🤣  Simply looks perfect ❤️ Great job 🙌

**Review by im-klau — APPROVED** [review]

**Pull request merged**